### PR TITLE
First stab on picture loading effect

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**

### CODE REVIEW

File: projects/storefrontlib/src/lib/ui/components/media/picture/picture.component.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
 img {
   max-width: 100%;
   max-height: 100%;
+
+  transform: scale(var(--cx-zoom, 1));
```
[the remark this answers is not in the document]
Comment (tobi-or-not-tobi) [reply projects/storefrontlib/src/lib/ui/components/media/picture/picture.component.scss]: I've done that slightly different in the component logic as the component is reused.

File: projects/storefrontlib/src/lib/ui/components/media/picture/picture.component.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ export class PictureComponent implements OnChanges {
   mainImage;
   missingProductImgSrc = missingProductImgSrc;
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/components/media/picture/picture.component.ts]: Just thinking here. There are many scenarios where this is used. Perhaps there could be an input so this feature can be switched off. The default could even be true but at least there is a way to do so.

**Comment by tobi-or-not-tobi** [comment]
@LTiger14 ready for re-review

File: projects/storefrontlib/src/lib/product/components/product-details/product-images/custom-styles.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Custom Styles
+
+## Vertical thumb panel
+
+The folowing snippets demonstrates the so-called amazon layout of thumbs. It's a small customisation that renders the thumbnails on the right hand side of the main image.
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-details/product-images/custom-styles.md]: The left hand side 😉

File: projects/storefrontlib/src/lib/product/components/product-details/product-images/product-images.component.ts [hunk 4]
```diff
@@ -8,19 +10,42 @@ import { Component, Input, OnChanges } from '@angular/core';
...
+    return imageContainer.zoom.url === this.mainImageContainer.zoom.url;
   }
 
-  isMainImage(image) {
-    return image.zoom.url === this.mainImage.zoom.url;
+  loadHandler() {
+    this.clearWaitList();
+  }
+
+  private start(el: HTMLElement) {
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-details/product-images/product-images.component.ts]: Could you just rename this to something more verbose like `startWaiting`

**Pull request merged**